pkg/transforms/policyreport.go

```diff
@@ -48,12 +48,25 @@ func PolicyReportResourceBuilder(pr *PolicyReport) *PolicyReportResource {
 	node.Properties["apiversion"] = gvk.Version
 	node.Properties["apigroup"] = gvk.Group
 
+	// Overwrite the name to be the namespace - which is same as the cluster the policy affects
+	node.Properties["name"] = pr.Namespace
```

Comment:
If we overwrite the name here, the UI will be broken when displaying the yaml. It will also break the delete action.

FYI @zlayne

Reply:
Yea lets just use the pr.Name (it will be the same as the namespace - we are using the cluster name for both). I am making the necessary changes in insights-client to change the name of policyreports to the cluster name/namespace.